package-lock.json

```diff
@@ -1084,7 +1084,6 @@
       "version": "6.12.2",
```

Comment:
Should we be merging all of these dependencies into master? (I think this is fine, but I just wanted to check)